test/test_manifest.json

```diff
@@ -706,6 +706,14 @@
        "lastPage": 1,
        "type": "eq"
     },
+    {  "id": "issue6289",
+       "file": "pdfs/issue6289.pdf",
+       "md5": "0869F3D147C734EC484FFD492104095D",
```

Comment:
This should be `0869f3d147c734ec484ffd492104095d` according to the test run logs (i.e., no capital letters).